### PR TITLE
Feature/init files

### PR DESCRIPTION
**Pull request by maosan132**
# Description

Hello pals, I am re-submitting this pull request so everybody can give support and suggestions for making this project better, so it can be approved. Any suggestions will be fully appreciated. 

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by trekab — COMMENTED** [review]
#### Invalid! :stop_sign: 
Hi @maosan132! Unfortunately, I have to invalidate your code review request because your current linter configuration won't run the linter checks as expected. Please redo your [linter configuration](https://github.com/microverseinc/linters-config/tree/master/javascript) and make sure that the necessary checks run on your pull request. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-))

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Comment by BrittanyBlake** [comment]
### Status: Invalid 🛑 
****
Hi @maosan132. This pull request has been marked as invalid because the linters are not configured correctly. Please take a look at how to set up the [linters config](https://github.com/microverseinc/linters-config/tree/master/javascript) and make the necessary changes.  


- It appears as though your project is missing the `.github` folder. Your linter config file should be in the`.github/workflows` folder.

- Also, please ensure that you have a descriptive `README` file before you submit your next pull request. It is missing a screenshot, the live demo link, and instructions on how to set up and run the project. 

****
Please make the required adjustments and submit a new code review request.

### Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Invalid :red_circle: 

- [ ]  This pull request will be marked as invalid in your project because GitHub Actions must be activated.
- [ ] It will be nice if you add screenshot in your readme as it is the first thing that users check.

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

File: src/fetch-data.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import { renderData, changeBackground } from './render-data';
+
+const flash = document.getElementById('flash'); // for showing errors
+
+async function fetchWeather(location) {
+  const api = '1627b8ae6ced3746531173abad9b4d06';
+  const unit = 'metric';
```
Comment (MuriloRoque) [inline src/fetch-data.js]: - [ ] You need to add an option to either make the calls on Fahrenheit or Celsius depending on the user's preferences or build a toggle button to switch between the two of them after getting the result ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png)).

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @maosan132 :wave: 

### Amazing on this project, your page is doing what it's supposed to do and it's truly beautiful, including the images you chose for each weather! Also, I really liked the way you structured your app, properly separating every file, congratulations! There's only one issue we need to solve before moving on, alright? Please check my comment in your code below :smile: 

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello again @maosan132 :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**